playbooks/robusta_playbooks/oom_killer.py

```diff
@@ -1,30 +1,48 @@
+from datetime import timedelta
+
 from robusta.api import *
-from robusta.integrations.kubernetes.api_client_utils import parse_kubernetes_datetime
-from datetime import datetime, timezone
-from collections import namedtuple
+from robusta.integrations.resource_analysis.memory_analyzer import MemoryAnalyzer, K8sMemoryTransformer
 
 
-def is_last_state_in_oom_status(status: ContainerStatus):
-    if not status.lastState:
-        return False
-    if not status.lastState.terminated:
-        return False
-    return status.lastState.terminated.reason == "OOMKilled"
+class OomKillerEnricherParams(ActionParams):
+    """
+    :var new_oom_kills_duration_in_sec: In order to avoid duplicated, This playbook will only report OOMKills
+    that occurred in the last new_oom_kills_duration_in_sec seconds.
+    For example, if new_oom_kills_duration_in_sec is 1200, only OOMKills from the last 20 minutes will be considered.
+    """
+    new_oom_kills_duration_in_sec: int = 1200
```

Comment:
These parameters are too internal for the common user.
We don't expect them to change that.
I think we should just take those from an env variable. 
maybe add this documentation just in the playbook's code, for a coding user

Reply:
I disagree here: I want to avoid the use of environment variables for configuration altogether (As it has no hierarchy), especially in the case of a playbooks configuration. The params are the configuration for the playbook, and it has a hierarchy and is automatically documented- so why use another mechanism?

Is the purpose here to avoid showing these "advanced" configurations in the docs? If so, I think a better way to address this would be to introduce something like an @advanced decorator for parameters and show the advanced parameter in the docs only if the user clicks "Advanced configuration"

Reply:
I agree scope wise it doesn't belong to env vars.
I just don't see any user ever changing these, cause it's really internal, and if that's the case, why exposing it.
dunno

Reply:
Mmm I see, is converting to CONSTS fine then?

Reply:
yeap, thx

Reply:
fixed